components/google_calendar/actions/create-calendar/create-calendar.js

```diff
@@ -0,0 +1,32 @@
+const googleCalendar = require("../../google_calendar.app");
+
+module.exports = {
+  key: "google_calendar_create_calendar",
+  name: "Create a Calendar in an user account",
+  description: "Create a Calendar in an user account.",
+  version: "0.0.1",
+  type: "action",
+  props: {
+    googleCalendar,
+    summary: {
+      type: "string",
+      description: "Title of the calendar.",
+      label: "Title",
+    },
+    description: {
+      type: "string",
+      description: "Description of the calendar.",
+      label: "Description",
+      optional: true,
+    },
+  },
+  async run() {
+    const calendar = this.googleCalendar.calendar();
+    return (await calendar.calendars.insert({
+      requestBody: {
+        summary: this.summary,
+        description: this.description,
+      },
+    })).data;
```

Comment:
I would recommend creating a new function named `insertCalendar` in `google_calendar.app.js` like this 
```javascript
    async insertCalendar(params) {
      const calendar = this.calendar();
      const { data } = await calendar.calendars.insert(params);
      return data;
    },
```
following the same pattern as the other functions. Then you can call the function like this
```javascript
    return this.googleCalendar.insertCalendar({
      requestBody: {
        summary: this.summary,
        description: this.description,
      },
    });
```